### ISSUE TITLE
My new layout for 4-panel comics.

### ISSUE THREAD

**Issue by WilliamPWise** [WilliamPWise/Prairie-World-Programming-Comics#9]
https://github.com/WilliamPWise/Prairie-World-Programming-Comics/blob/master/Pencils/P09-Variables.jpg

ttps://github.com/WilliamPWise/Prairie-World-Programming-Comics/blob/master/Pencils/P10-Operators.jpg

I have upload a couple of pencil drafts on works to replace Variables and Operators. This displays a new style for my 4 panel layout, with the following features:

-The text is shrunk down a bit and uses more even spacing

-There is more padding around the borders to control the image density.

-It goes top-down for easier mobile viewing, and so that two 4-panels can be easier to print on one page.

Hopefully, it works better than my last layout. Tell me what you think of it!

**Comment by tutordelphia** [comment]
Huge improvement! I particularly like the Variables one.

I'm having a bit of trouble with the following wording: 

> The "reference values" operators derive from are known as Operands:

There may be a grammatical error there but also, who is your audience for these comics? You try to explain concepts in simple, easy to understand ways. I'm not sure that sentence fits with that. 

Otherwise, those two look great!

**Comment by tutordelphia** [comment]
Hi William, while I think that's better, what I think about it doesn't matter. You need to find a member of your target audience and ask them. If you don't know who your target audience is or can't find a single member of it, then that may be a bigger problem that you need to solve first (but I think you are creating an appealing project so I think you won't have trouble). I think you are on the right track and making improvements. I've just been reading a lot lately about the success and failure of projects and client/target audience involvement. I'll post on Code Newbies to see if anyone there is interested.

**Comment by WilliamPWise** [comment]
I've never heard of Code Newbies before. Let me see if I can find the site.

**Comment by WilliamPWise** [comment]
So I found this site called "CodeNewbie". Is this the one? Somehow, this organization branches out into so many areas, and I don't know which area you speak of.

**Comment by tutordelphia** [comment]
Yes, https://www.codenewbie.org/ . I asked on the Slack channel but didn't get any replies. They also have a project BaseCS which covers coding topics with a podcast and blog posts with hand-drawn illustrations. There is also a Twitter chat held weekly which discusses topics talked about on their main podcast. But my point wasn't to focus on them (some people there are true newbies and others just stay because it is a very supportive community) so much as finding people who are your target audience and making sure you are getting feedback from them rather than just from people who already know the topics.

**Comment by WilliamPWise** [comment]
From the Readme, my target audience is the general audience of newcomer and aspiring programmers. It isn't really young children - I just want my work to be accessible to as many people as possible.

I don't care for niches. I just want my work to be great at explaining concepts to people who don't know them. Yes, it is a broad spectrum, but that's what I want my work to be for.

**Comment by WilliamPWise** [comment]
Also - I'd gladly accept feedback from newcomers. The issue I have is that I'm no expert - and I rarely have ever written code. In this regard it feels like a 'blind leading the blind' situation. One example as a result:

https://www.reddit.com/r/webcomics/comments/7zsb4m/programming_elements_variables/

This was the original draft about variables, and watch where I didn't have enough knowledge about variables. My discussion about the subject here ended up just confusing the newcomer a lot more. I know more now, but it was only through discussion with experts could I explain it accurately.
